boards/OPENMV_AE3/omv_boardconfig.h

```diff
@@ -112,6 +115,7 @@ extern unsigned char OMV_BOARD_UID_ADDR[12];    // Unique address.
 #define OMV_OPENAMP_SIZE                (64K)
 #define OMV_GC_BLOCK0_MEMORY            SRAM0   // Extra GC block 1
 #define OMV_GC_BLOCK0_SIZE              (4M)
+#define OMV_FAST_MEMORY                 ".data" // Fast memory for quad port reads/writes.
```

Comment:
That's not how you add those, but the AE3 does Not even need this, main memory is already in DTCM, so let's just remove it.